Selenium.Axe/Selenium.Axe/EmbeddedResourceAxeProvider.cs

```diff
@@ -1,9 +1,17 @@
-﻿using Selenium.Axe.Properties;
+﻿using System.IO;
+using System.Reflection;
+using System.Text;
 
 namespace Selenium.Axe
 {
     internal class EmbeddedResourceAxeProvider : IAxeScriptProvider
     {
-        public string GetScript() => Resources.axe_min;
+        public string GetScript() {
+            var assembly = Assembly.GetAssembly(typeof(EmbeddedResourceAxeProvider));
+            var resourceStream = assembly.GetManifestResourceStream("Selenium.Axe.Resources.axe.min.js");
+            using (var reader = new StreamReader(resourceStream, Encoding.UTF8)) {
+                return reader.ReadToEnd();
+            }
```

Comment:
might be better to make this generic. may be have a class EmbededResourceProvider.GetResourceContent(resourceName).

& let this class use that.